src/Illuminate/Support/Collection.php

```diff
@@ -1124,10 +1124,10 @@ public function take($limit)
     public function drop($limit)
     {
         if ($limit < 0) {
-            $limit = abs($limit);
+            return $this->slice(0, $limit);
```

Comment:
Couldn't the same be said for `take` and `forPage` as well? They too are only providing syntactic sugar around slice calls. Especially since the behavior of `slice` changes quite a bit between positive and negative parameters. So I don't think it's too unreasonable to introduce a more expressive method for dropping elements from a collection. I much prefer `$collection->drop(-2)` over `$collection->slice(0, -2)`.